File: src/main/scala/resonantinduction/archaic/waila/WailaRegistrar.java
package resonantinduction.archaic.waila;

import resonantinduction.archaic.crate.TileCrate;
import resonantinduction.archaic.fluid.tank.TileTank;
import mcp.mobius.waila.api.IWailaRegistrar;

public class WailaRegistrar
{
    public static void wailaCallBack(IWailaRegistrar registrar)
    {
        registrar.registerBodyProvider(new WailaCrate(), TileCrate.class);
        registrar.registerBodyProvider(new WailaFluidTank(), TileTank.class);
    }
}

File: src/main/scala/resonantinduction/archaic/waila/WailaRegistrar.scala
package resonantinduction.archaic.waila

import mcp.mobius.waila.api.IWailaRegistrar
import resonantinduction.archaic.crate.TileCrate
import resonantinduction.archaic.fluid.tank.TileTank

object WailaRegistrar
{
    def wailaCallBack(registrar: IWailaRegistrar)
    {
        registrar.registerBodyProvider(new WailaCrate, classOf[TileCrate])
        registrar.registerBodyProvider(new WailaFluidTank, classOf[TileTank])
    }
}